src/contexts/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface IVoteArgs {
```

Comment:
Let's add new line at the end of the file and there are two more interfaces left